docs/configuration.md

```diff
@@ -4672,6 +4672,12 @@ The parameter also allows the following shortcuts:
 
 ### `spaces.beforeContextBoundColon`
 
+This parameter controls if a space should be used before a colon that precedes
```

Comment:
Do we need a separate setting here?

Reply:
i am planning to use this same set of values for spaces within braces, and it makes little sense to add those spaces after looking at some other bound which is not within those braces. or to add a completely new set of values which are likely named the same but carry a slightly different meaning. just for consistency.

Reply:
Ok, thanks for explaining!